core/src/main/java/com/amplifyframework/datastore/DataStoreCategoryBehavior.java

```diff
@@ -97,6 +97,22 @@
             @NonNull Consumer<DataStoreException> onFailureToDelete
     );
 
+    /**
+     * Deletes every item of given type from the DataStore that meets the provided
+     * conditions.
```

Comment:
What's the behavior if there are no matches? Nothing gets deleted? Can you document?

Reply:
Nothing is deleted and operation succeeds. I will add it to the doc.